client/me/account/main.jsx

```diff
@@ -570,12 +570,12 @@ const Account = createReactClass( {
 				</FormFieldset>
 
 				<FormFieldset>
-					<FormLabel htmlFor="primary_site_ID">{ translate( 'Primary Site' ) }</FormLabel>
+					<FormLabel htmlFor="primary_site_ID">{ translate( 'Primary site' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.